test/unit/doc_result_test.rb

```diff
@@ -9,7 +9,7 @@ class DocResultTest < ActiveSupport::TestCase
       doc_type: 'QUESTIONS',
       doc_url: question.path(:question),
       doc_title: question.title,
-      score: question.answers.length
+      # score: question.answers.length    What should I do about this particular field?
```

Comment:
For running locally, I've commented this out. But that isn't the right way to do it. What exactly is score? Should I replace this with the number of comments on a particular question?

Reply:
Yes, comments sounds great! Answers were migrated to be comments.